paddle/fluid/framework/parallel_executor.cc

```diff
@@ -63,7 +64,17 @@ ParallelExecutor::ParallelExecutor(
   member_->global_scope_ = scope;
   member_->use_cuda_ = exec_strategy.use_cuda_;
 
-  // Step 1. Bcast the params to devs.
+  // Step 1. check all memory usages of all places.
```

Comment:
So these code is not needed?

Reply:
I think these messages should be print out also, so I didn't remove it.